.github/workflows/installation-tests.yml

```diff
@@ -138,19 +138,19 @@ jobs:
           run: docker exec -i pki.example.com ${PKIDIR}/ci/ds-create.sh
 
         - name: Install CA
-          run: docker exec -i pki.example.com ${PKIDIR}/ci/ca-create.sh
+          run: docker exec -i pki.example.com pkispawn -f ${PKIDIR}/base/server/examples/installation/ca.cfg -s CA -v
```

Comment:
Question: Do we want to specify the full path?( `/usr/sbin/pkispawn` rather than `pkispawn`?)

Reply:
I think it's better without the full path because that way the test will not be Fedora-specific and it will actually validate that `pkispawn` was installed in the correct place.